### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by Anmol6** [Michael-F-Bryan/auto-changelog#101]
Can we get a new release tag soon? The master build seems to be much better than 0.5.1
Thanks!

**Comment by LeMimit** [comment]
I agree on this one. There are:
- Cool new option: `--template`
- Some major bug fix in the parsing of the commit message: [`582beb3`](https://github.com/Michael-F-Bryan/auto-changelog/commit/582beb3ceb8332301903f1c598c2ed45b044a6d8), [`f395f87`](https://github.com/Michael-F-Bryan/auto-changelog/commit/f395f878af372106c8a27bb044944182c93d59d1), [`c95799`](https://github.com/Michael-F-Bryan/auto-changelog/commit/c957992564d4f2e3610cdba6969e9c9b30f0d016) + other bug fixe
- A debug mode

These change will probably/may be fix #75.

Enough content to do a `0.6.0` ? @KeNaCo

**Comment by KeNaCo** [comment]
I agree, sorry for the inactivity (covid+family+work). But now the majority of issues are settled and I'm going to invest in the project one evening per week.

That said, expect 0.6.0 released next weekend _(unless I spot some major issue)._
